src/components/site/PageHeader.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+const PageHeader = props => {
+  const { title, preTitle, deck, backGroundImage, className } = props;
+  const headerCssClasses = classnames("dg_page-header", className);
+  return (
+    <div className={headerCssClasses}>
+      <img src={backGroundImage} />
+      <div className="dg_page-header-top-left">
+        <div className="row">
+          <div className="col-12 col-lg-5 col-xl-6">
+            <span className="dg_page-header__pre-title">
+              {preTitle !== undefined ? preTitle : null}
+            </span>
+            <span className="dg_page-header__title">{title} </span>
+          </div>
+          <div className="col-12 col-lg-7 col-xl-6">
+            <p>{deck !== undefined ? deck : null}</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+PageHeader.propTypes = {
+  /** className: Displays transparent color on the top of the image */
+  className: PropTypes.string,
```

Comment:
you dont need to add thsi here, it className is an available prop on all components.

Reply:
done